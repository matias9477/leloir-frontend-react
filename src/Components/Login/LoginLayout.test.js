import React from 'react';
import { shallow } from 'enzyme';

import LoginForm from './LoginLayout';

describe('LoginForm', () => {
    it('should render correctly in "debug" mode', () =>{
        const component = shallow(<LoginForm debug />);

        expect(component).toMatchSnapshot();
    });
});
