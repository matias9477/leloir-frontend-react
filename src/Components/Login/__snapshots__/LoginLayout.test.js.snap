// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm should render correctly in "debug" mode 1`] = `
<div
  className="loginLayout"
>
  <Segment>
    <Image
      as="img"
      centered={true}
      size="small"
      src="/android-chrome-192x192.png"
      ui={true}
    />
    <withRouter(Connect(LoginComponent)) />
  </Segment>
</div>
`;
